### PR TITLE
Added support for linux non wsl

### PR DESCRIPTION
**Pull request by CapedBojji**
I added support for linux, using xdg-open

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
-		"build": "tspc",
-		"build-watch": "tspc -w",
+		"build": "tsc",
+		"build-watch": "tsc -w",
 		"prepublishOnly": "npm run build",
 		"devlink": "cd devlink && npm link"
 	},
 	"keywords": [],
-	"author": "osyrisrblx",
+	"author": "CapedBojji",
```
Comment (christopher-buss) [inline package.json:16]: You've changed the autor here.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
 .DS_Store
 node_modules
```
Comment (christopher-buss) [inline .gitignore:2]: Is there a reason you've removed the out folder from gitignore? This should all be autogenerated.

File: package.json (hunk 1 above)
Comment (CapedBojji) [reply package.json:16]: My bad you qere taking long so i want gonna upload a linuc version called rbxts-build-linux. Will fix that

File: .gitignore (hunk 2 above)
Comment (CapedBojji) [reply .gitignore:2]: I needed to use it, i was pulling it with npm with a direct link toto my github thuse i needed the compiled version sorry. Wil add that also

**Pull request closed without merging**